packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -81,6 +81,13 @@ class InflationManager {
     const parametersPost = await this.getParameters()
     assertEqualBN(parametersPost.factor, ONE)
   }
+
+  setGasCurrencyCost = async (cost: number) => {
```

Comment:
Nit: `InflationManager` doesn't seem like the right place for this to live in - I don't see a direct relation to inflation.